src/pages/Login/Login.js

```diff
@@ -1,9 +1,80 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Input from './Input/Input';
+import Button from './Button/Button';
+import SignUp from './SignUp/SignUp';
+import Outer from './SignUp/Outer';
 import './Login.scss';
 
 class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      isButtonOn: false,
+    };
+  }
+
+  getInputValue = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleButton = e => {
+    const { id, pw } = this.state;
+    (id.length < 4 || id.length > 12) && pw.length > 5
+      ? this.setState({ isButtonOn: true })
+      : this.setState({ isButtonOn: false });
+  };
```

Comment:
```suggestion
  handleButton = e => {
    const { id, pw } = this.state;
    const isButtonOn = (id.length < 4 || id.length > 12) && pw.length > 5
    
    this.setState({ isButtonOn })
  };
```
- 판별 식에 변수명을 붙여 설명해주면 좋을거 같습니다
- 변수명과 객체의 키값이 동일할 경우에는 축약 가능합니다

Reply:
이 부분 기능 수정완료했습니돠!!!